regions/Switzerland/scripts/helpers.py

```diff
@@ -162,72 +62,28 @@ def get_cmap_hex(cmap, length):
 
     return hex_codes
 
+def emptyfolder(path):
+    """Removes all files and subdirectories in the given folder."""
+    if os.path.exists(path):
+        for item in os.listdir(path):
+            item_path = os.path.join(path, item)
+            try:
+                if os.path.isfile(item_path):
+                    os.remove(item_path)  # Remove file
+                elif os.path.isdir(item_path):
+                    shutil.rmtree(item_path)  # Remove folder and all contents
+            except Exception as e:
+                print(f"Error removing {item_path}: {e}")
+    else:
+        os.makedirs(path, exist_ok=True)  # Ensure directory exists
 
-def free_up_cuda():
-    gc.collect()
-    torch.cuda.empty_cache()
-
-
-def makeCombNum(combin_clim, combi_topo):
-    topo_primes = {
-        'aspect': 2,
-        'slope': 3,
-        'dis_from_border': 5,
-        'topo': 7,
-    }
-    climate_primes = {
-        't2m': 11,
-        'tp': 13,
-        'slhf': 17,
-        'sshf': 19,
-        'ssrd': 23,
-        'fal': 29,
-        'str': 31,
-    }
-    mult = 1
-    for var in combi_topo:
-        mult *= topo_primes[var]
-    for var in combin_clim:
-        mult *= climate_primes[var]
-    return mult
-
-
-# Get all  combinations of length min 3:
-def powerset(original_list, min_length=3):
-    # The number of subsets is 2^n
-    num_subsets = 2**len(original_list)
-
-    # Create an empty list to hold all the subsets
-    subsets = []
-
-    # Iterate over all possible subsets
-    for subset_index in range(num_subsets):
-        # Create an empty list to hold the current subset
-        subset = []
-        # Iterate over all elements in the original list
-        for index in range(len(original_list)):
-            # Check if index bit is set in subset_index
-            if (subset_index & (1 << index)) != 0:
-                # If the bit is set, add the element at this index to the current subset
-                subset.append(original_list[index])
-        # Add the current subset to the list of all subsets
-        if len(subset) >= min_length:
-            subsets.append(subset)
-    return subsets
-
+# difference between two lists
+def Diff(li1, li2):
+    li_dif = [i for i in li1 + li2 if i not in li1 or i not in li2]
```

Comment:
If the order isn't important, this is a little more Pythonic :)
```suggestion
    li_dif = list(set(li1) - set(li2))
```